src/Stripe.net/Services/Invoices/InvoiceUpdateOptions.cs

```diff
@@ -32,6 +32,12 @@ public class InvoiceUpdateOptions : BaseOptions
         [JsonProperty("days_until_due")]
         public long? DaysUntilDue { get; set; }
 
+        /// <summary>
+        /// ID of the default payment method for the invoice.
+        /// </summary>
+        [JsonProperty("custom_fields")]
+        public string DefaultPaymentMethodId { get; set; }
+
```

Comment:
Same as in Go. We missed `payment_method` in https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Invoices/InvoicePayOptions.cs#L8